aspnetcore-features/CustomAuthorizeAttribute/CustomAuthorizeAttribute/Controllers/WeatherForecastController.cs

```diff
@@ -0,0 +1,47 @@
+using Microsoft.AspNetCore.Authorization;
+using Microsoft.AspNetCore.Mvc;
+
+namespace CustomAuthorizeAttribute.Controllers;
+
+[ApiController]
+[Route("[controller]")]
+public class WeatherForecastController : ControllerBase
+{
+    private static readonly string[] Summaries = new[]
+    {
+        "Freezing", "Bracing", "Chilly", "Cool", "Mild", "Warm", "Balmy", "Hot", "Sweltering", "Scorching"
+    };
+
+    private readonly ILogger<WeatherForecastController> _logger;
```

Comment:
I just wanted to check this logger was autogenerated from the boilerplate. If yes, leave it as is. Else please remove it as the _logger is not used anywhere.

Reply:
Yes, it's autogenerated.